src/control-flow-basics/loops/for.md

```diff
@@ -1,19 +1,25 @@
 # `for`
 
 The [`for` loop](https://doc.rust-lang.org/std/keyword.for.html) iterates over
-ranges of values:
+ranges of values or the items in a collection:
 
 ```rust,editable
 fn main() {
     for x in 1..5 {
         println!("x: {x}");
     }
+
+    for elem in [1, 2, 3, 4, 5] {
+        println!("elem: {elem}");
+    }
```

Comment:
I think this was part of the strategy of only using value types at first.

However, I can see we break this again in [6.3.1. Labels](https://google.github.io/comprehensive-rust/control-flow-basics/break-continue/labels.html) where we use more arrays.

I agree that the syntax is simple and understandable. I would be inclined to remove the `1..5` range syntax since I find it more complex: as a new person, I would wonder what this becomes and if the `..` part is part of `for`-loop syntax (it's not).

Reply:
Thanks, I asked Dmytro as well and he also likes it!